src/main/resources/application-deployment.yaml

```diff
@@ -0,0 +1,4 @@
+server:
```

Comment:
Profile file usually [has name as environment name](https://www.baeldung.com/spring-profiles#2-profile-specific-properties-files). Environments mean separate machine (like hiroky containers)  for main branches (master, development, release, ect.) For example, branch `development` is deployed on dev environment with appropriate configuration.
So please change name
application-development.yaml or application-dev.yaml

Reply:
done